keps/prod-readiness/sig-auth/541.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 541
```

Comment:
@annajung - continuing conversation[0] about updating this kep for 1.21, I added this `keps/prod-readiness/...` file per your note. Let me know if this is not what we need.

[0] https://kubernetes.slack.com/archives/C0EN96KUY/p1611762222015600?thread_ts=1611606235.003600&cid=C0EN96KUY

Reply:
perfect, this is exactly what's needed as noted in the [PRR requirement](https://github.com/kubernetes/community/blob/master/sig-architecture/production-readiness.md#submitting-a-kep-for-production-readiness-approval), thanks!